### ISSUE TITLE
Missing files in sdist

### ISSUE THREAD

**Issue by thatch** [swimos/swim-system-python#50]
It appears that the manifest is missing at least one file necessary to build
from the sdist for version 1.0.0a0. You're in good company, about 5% of other
projects updated in the last year are also missing files.

```
+ /tmp/venv/bin/pip3 wheel --no-binary swimai -w /tmp/ext swimai==1.0.0a0
Looking in indexes: http://10.10.0.139:9191/root/pypi/+simple/
Collecting swimai==1.0.0a0
  Downloading http://10.10.0.139:9191/root/pypi/%2Bf/f48/eb5f80f51d210/swimai-1.0.0a0.tar.gz (60 kB)
    ERROR: Command errored out with exit status 1:
     command: /tmp/venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-wheel-6pzku6q_/swimai/setup.py'"'"'; __file__='"'"'/tmp/pip-wheel-6pzku6q_/swimai/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-wheel-6pzku6q_/swimai/pip-egg-info
         cwd: /tmp/pip-wheel-6pzku6q_/swimai/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-wheel-6pzku6q_/swimai/setup.py", line 20, in <module>
        with open('requirements.txt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by DobromirM** [comment]
Thank you for reporting this issue! It should now be resolved in version 1.0.1-alpha.

**Comment by thatch** [comment]
Confirmed, and thanks!
